Support Celo

aminlatifi opened the pull request.

aminlatifi committed:
Upgraded monoswap to v1.3.0

aminlatifi committed:
Added CELO and CELO ALFAJORES support

aminlatifi committed:
Updated example config comment of default values of celo and celo alfajores networks rpc urls

aminlatifi committed:
Completed tests of CELO
Fixed issue on configuring optimism api default endpoint

aminlatifi committed:
Added INFURA_ID and other env keys to github action tests

mohammadranjbarz commented on migration/1680014857601-addCeloTokens.ts:
```diff
@@ -0,0 +1,41 @@
+import { NETWORK_IDS } from '../src/provider';
+
+// tslint:disable-next-line:class-name
+export class addGoerliTokens1677742523974 implements MigrationInterface {
```
@aminlatifi  Please change name of the class file

mohammadranjbarz commented on migration/1680014857601-addCeloTokens.ts:
```diff
@@ -0,0 +1,41 @@
+    );
+    const tokens = await queryRunner.query(`
+            SELECT * FROM token
+            WHERE "networkId" = ${NETWORK_IDS.CELO} or "networkId" = ${NETWORK_IDS.CELO_ALFAJORES}
```
I think you should set some condition here and don't `CELO_ALFAJORES` tokens in Production ENV ( I assumed this is test network of Celo)

aminlatifi committed:
Added more networks to transaction network id config of admin bro
Changed wrong names of migration files made by copying
Only add celo token to production env and celo alfajores to the test env

mohammadranjbarz reviewed (approved):
Thanks @aminlatifi  LGTM
